diffkemp/simpll/Utils.cpp

```diff
@@ -50,25 +51,47 @@ static std::vector<Attribute::AttrKind> badVoidAttributes = {
         Attribute::AttrKind::Dereferenceable,
         Attribute::AttrKind::DereferenceableOrNull};
 
-/// Extract called function from a called value Handles situation when the
-/// called value is a bitcast.
-const Function *getCalledFunction(const Value *CalledValue) {
-    const Function *fun = dyn_cast<Function>(CalledValue);
+/// Convert a value to a function.
+/// Handles situation then the actual function is inside a bitcast or alias.
+const Function *valueToFunction(const Value *Value) {
+    auto *fun = dyn_cast<Function>(Value);
```

Comment:
I believe that `Fun` would be better here, since other variable names are also capitalized.